src/entity/Action.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
+
+export enum ActionType {
+  Notification = 'notification',
+}
+
+@Entity()
+export class Action {
```

Comment:
We should consider a more explicit name. `Action` is too generic I believe.
@rebelchris @sshanzel thoughts?

Reply:
What do you both think of `CompletedAction`?

Reply:
Perhaps `UserAction` or `OnboardingAction` idk. But something with more context as who and maybe why

Reply:
This should have been noticed on the DR for future reference.
Let's go with UserAction as it's not particular to onboarding actions it can be used for more things.

Reply:
@rebelchris agree, my bad :)